packages/drivers/odsp-driver/src/odspDriverUrlResolver.ts

```diff
@@ -50,7 +50,73 @@ export class OdspDriverUrlResolver implements IUrlResolver {
     constructor() { }
 
     public async resolve(request: IRequest): Promise<IOdspResolvedUrl> {
-        return resolveRequest(request);
+        if (request.headers && request.headers[DriverHeader.createNew]) {
```

Comment:
Why do we need this code path? I thought DriverHeader.createNew is no longer used anywhere and should be removed in favor or detached container flow.

Reply:
DriverHeader.createNew is the detached container path.